tests/lib/cli-engine/cli-engine.js

```diff
@@ -1774,7 +1772,7 @@ describe("CLIEngine", () => {
             engine = new CLIEngine({
                 cwd: originalDir,
                 useEslintrc: false,
-                rules: { indent: 1, "valid-jsdoc": 0, "require-jsdoc": 0 }
+                rules: { indent: 1, "callback-return": 0 }
```

Comment:
This line is causing merge conflicts. The fixed line should be `rules: { eqeqeq: 1, "callback-return": 0 }` (`indent` was replaced here with `eqeqeq` in https://github.com/eslint/eslint/pull/17696).